From 61ea05f315532faaef44caf48963ec69efbd573d Mon Sep 17 00:00:00 2001
From: Luke Bennett <lukeeeebennettplus@gmail.com>
Date: Thu, 11 Jul 2019 17:08:32 +0100
Subject: [PATCH 75/84] Update postcss-initial to 3.0.1

postcss-initial@3.0.1 fixes a proto polution vulnerability in its
lodash.template dependency.
---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 9f72176..b6d6029 100644
--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
     "postcss-font-variant": "^4.0.0",
     "postcss-gap-properties": "^2.0.0",
     "postcss-image-set-function": "^3.0.1",
-    "postcss-initial": "^3.0.0",
+    "postcss-initial": "^3.0.1",
     "postcss-lab-function": "^2.0.1",
     "postcss-logical": "^3.0.0",
     "postcss-media-minmax": "^4.0.0",
-- 
2.33.1

